core/src/main/java/org/javarosa/core/model/FormDef.java

```diff
@@ -1333,13 +1333,14 @@ public void populateDynamicChoices(ItemsetBinding itemset, TreeReference curQRef
                 new EvaluationContext(exprEvalContext, itemset.contextRef.contextualize(curQRef)));
 
         if (matches == null) {
-            throw new XPathException("Could not find references depended on by " + itemset.nodesetRef.getInstanceName());
+            throw new XPathException("No items found at '" + itemset.nodesetRef +
```

Comment:
@phillipm Waiting on this change to merge (or let me know if there's a reason you don't want to make the change)